src/peft/tuners/lora/layer.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, base_layer: nn.Module, **kwargs) -> None:
         # Mark the weight as unmerged
         self._disable_adapters = False
         self.merged_adapters = []
+        self.use_dora: dict[str, bool] = {}
+        self.lora_magnitude_vector: torch.nn.ParameterDict = torch.nn.ParameterDict({})  # for DoRA
```

Comment:
Hmmm this means now for all LoRA models we save this extra param. in the state dict (even if it's empty), do you know if that's the case? If not i think it should be all good, otherwise I think we should init it to None - what do you think?

Reply:
I think it's not included in the `state_dict`. E.g. we already have `lora_A` vs `lora_embedding_A` etc., one of which is always empty depending on layer type. But I still changed the default to be `None`, as it makes the repr smaller.